cli/src/pcluster/cli_commands/commands.py

```diff
@@ -243,152 +159,6 @@ def ssh(args, extra_args):
-            )
-            if not verified:
-                result = PclusterApi().describe_cluster(cluster_name=args.cluster_name, region=get_region())
-                if isinstance(result, ClusterInfo):
-                    utils.log_stack_failure_recursive(result.stack_arn, failed_states=["DELETE_FAILED"])
-                elif isinstance(result, ApiFailure):
-                    # If stack is already deleted
-                    if f"Cluster {args.cluster_name} doesn't exist." in result.message:
-                        LOGGER.warning("\nCluster %s has already been deleted or does not exist.", args.cluster_name)
-                        sys.exit(0)
-                    LOGGER.critical(result.message)
-                    sys.stdout.flush()
-                    sys.exit(1)
-                else:
-                    utils.error(f"Unable to retrieve the status of the cluster.\n{result.message}")
-
-            sys.stdout.write("\rStatus: %s\n" % result.stack_status)
-            sys.stdout.flush()
-            LOGGER.debug("Status: %s", result.stack_status)
-        else:
-            sys.stdout.write("\n")
-            sys.stdout.flush()
-        if result.stack_status == "DELETE_FAILED":
-            LOGGER.info("Cluster did not delete successfully. Run 'pcluster delete %s' again", args.cluster_name)
-    except ClientError as e:
-        if e.response.get("Error").get("Message").endswith("doesn't exist"):
-            LOGGER.warning("\nCluster %s has already been deleted or does not exist.", args.cluster_name)
-            sys.exit(0)
-        LOGGER.critical(e.response.get("Error").get("Message"))
-        sys.stdout.flush()
-        sys.exit(1)
-    except KeyboardInterrupt:
-        LOGGER.info("\nExiting...")
-        sys.exit(0)
-
-
-def start(args):
-    """Start cluster compute fleet."""
-    try:
-        result = PclusterApi().update_compute_fleet_status(
-            cluster_name=args.cluster_name,
-            region=get_region(),
-            status=ComputeFleetStatus.START_REQUESTED,
-        )
-        if isinstance(result, ApiFailure):
-            utils.error(f"Unable to start the compute fleet of the cluster.\n{result.message}")
-        else:
-            LOGGER.info("Compute fleet started correctly.")
-    except KeyboardInterrupt:
-        LOGGER.info("Exiting...")
-        sys.exit(0)
-
-
-def stop(args):
-    """Stop cluster compute fleet."""
-    try:
-        result = PclusterApi().update_compute_fleet_status(
-            cluster_name=args.cluster_name,
-            region=get_region(),
-            status=ComputeFleetStatus.STOP_REQUESTED,
-        )
-        if isinstance(result, ApiFailure):
-            utils.error(f"Unable to stop the compute fleet of the cluster.\n{result.message}")
-        else:
-            LOGGER.info("Compute fleet stopped correctly.")
-    except KeyboardInterrupt:
-        LOGGER.info("Exiting...")
-        sys.exit(0)
-
-
 def build_image(args):
```

Comment:
do we need to keep this?

Reply:
You're just referring to the try / except?